backend/src/EventSubscriber/JWTCreatedSubscriber.php

```diff
@@ -60,4 +77,17 @@ private function buildRoomPermission(User $user): string
 
         return '';
     }
+
+    private function getRoomFromRequest(User $user): ?string
+    {
+        if (null === $currentRequest = $this->requestStack->getCurrentRequest()) {
```

Comment:
```suggestion
        $currentRequest = $this->requestStack->getCurrentRequest();

        if (null === $currentRequest) {
```

This could apply to more ifs in this code, we should avoid doing an assignmnent and an if on the same statement, it is harder to read.